menu/drivers/ozone/ozone_sidebar.c

```diff
@@ -686,6 +686,7 @@ void ozone_context_reset_horizontal_list(ozone_handle_t *ozone)
             and the subsequent space */
          chr          = title_noext;
          hyphen_found = false;
+         settings_t *settings = config_get_ptr();
```

Comment:
Ah yes, there's also a build error here. Variables need to be declared before any other statements within each scope. So `settings_t *settings = config_get_ptr();` should be moved to line 576. It's always best to handle the case where `settings` is `NULL`, as well.  :)

Reply:
Thanks a lot for pointing out these mistakes! I will definitely work on this today.